src/storage/wal_replay.cpp

```diff
@@ -409,6 +418,32 @@ void ReplayState::ReplayDropMacro() {
 	catalog.DropEntry(context, &info);
 }
 
+//===--------------------------------------------------------------------===//
+// Replay Table Macro
+//===--------------------------------------------------------------------===//
+void ReplayState::ReplayCreateTableMacro() {
+	auto entry = ScalarMacroCatalogEntry::Deserialize(source);
```

Comment:
This is still not tested, and this actually seems incorrect (shouldn't this be `TableMacroCatalogEntry::Deserialize`)?

Reply:
Well spotted mark !!
Have corrected this